core/base/PeriodicImplicitTriangulation/PeriodicImplicitTriangulation.h

```diff
@@ -0,0 +1,13204 @@
+#ifndef _PERIODICIMPLICITTRIANGULATION_H
```

Comment:
stupid question: does **all** the API support periodicity? (or just a selection of the functions)

Reply:
All functions are re-written to support periodicity.